test/SecTester.Scan.Tests/Models/ModelEnumJsonSerializationTests.cs

```diff
@@ -0,0 +1,152 @@
+using SecTester.Bus.Dispatchers;
+using Module = SecTester.Scan.Models.Module;
+
+namespace SecTester.Scan.Tests.Models;
+
+public class JsonStringEnumMemberConverterTests
```

Comment:
The test suite name and SUT are different, move all of them to `DefaultMessageSerializerTests`